iml-gui/crate/src/generated/mod.rs

```diff
@@ -0,0 +1,3 @@
+// File `css_classes.rs` is (re)created during webpack compilation.
```

Comment:
Should this file be committed at all?

Reply:
I think so. Without this file, we couldn't run `cargo check` on this crate without errors.

Likewise, it is generated by webpack and we probably shouldn't be required to run that every time for checking the crate.